composer.json

```diff
@@ -52,7 +52,9 @@
         "monolog/monolog": "~1.11",
         "phpunit/phpunit": "~4.5",
         "satooshi/php-coveralls": "~0.6",
-        "symfony/phpunit-bridge": "~2.8|~3.0"
+        "symfony/phpunit-bridge": "~2.8|~3.0",
+        "doctrine/orm": "^2.4.8",
```

Comment:
Why do we require `doctrine/orm` if the code is also supposed to work without ORM?